### DISCUSSION TITLE
Netlify CMS

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#3] [Development (Open discussions)]
Use  [Netlify CMS](https://www.netlifycms.org) for the non-coders to contribute to the project without having to know how to code or how `git` works.

**Comment by shawnthompson** [comment]
@duboisp Would you like to help out with this?

**Reply by duboisp** [reply]
Yes, I will contact you to book a time

**Reply by duboisp** [reply]
We can follow the instruction here:
https://www.netlifycms.org/docs/add-to-your-site/

1. We would need to create an "admin" folder in our repository here: "_site"
2. Create an `_site/admin/index.html` file
```html
<!doctype html>
<html>
<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Content Manager for gc-da11yn</title>
</head>
<body>
  <!-- Include the script that builds the page and powers Netlify CMS -->
  <script src="https://unpkg.com/netlify-cms@^2.0.0/dist/netlify-cms.js"></script>
</body>
</html>
```
3. Create an `_site/admin/config.yml` file
```yml
backend:
  name: git-gateway
  branch: master # Branch to update
publish_mode: editorial_workflow
```
4. Configure Auth 
    * Step by step guide: https://www.netlify.com/blog/2016/10/27/a-step-by-step-guide-deploying-a-static-site-or-single-page-app/
    * Ref. https://www.netlifycms.org/docs/add-to-your-site/#enable-identity-and-git-gateway
5. Add Identity widget
    * Ref. https://www.netlifycms.org/docs/add-to-your-site/#add-the-netlify-identity-widget
6. We should be good to go with editing pages

**Comment by duboisp** [comment]
@shawnthompson we will need to check if gc-da11yn.github.io qualify as per the following Open Source Plan Policy for freely using Netlify: https://www.netlify.com/legal/open-source-policy/?_ga=2.136268657.1506282937.1637814624-213133609.1613020368

Self-hosted GitHub support are only available for the Business plan, not for the Pro plan neither for the free plan.

What about using [prose.io](https://github.com/prose/prose/wiki/Getting-Started) instead? It is free and designed to work with github pages. Or even simpler, let use github built-in text editor and explain to our contributor how to write markdown code and how to preview it.
